samples/nrf9160/lwm2m_client/src/config.h

```diff
@@ -5,8 +5,10 @@
  */
 
 #if defined(CONFIG_LWM2M_DTLS_SUPPORT)
-static char client_psk[] = CONFIG_APP_LWM2M_PSK;
 
-#define SERVER_TLS_TAG CONFIG_APP_SERVER_TLS_TAG
```

Comment:
As already pointed out, I don't see a reason to remove existing configs.

Reply:
Fixed